circle.yml

```diff
@@ -13,4 +14,5 @@ deployment:
   prod:
     branch: master
     commands:
+      - aws s3 sync jekyll/_staging_site s3://circle-staging-docs --delete
```

Comment:
as i mentioned in the other pr, i'd use a bucket that is itselft a viable hostname, then you can just use static hosting.